From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "J. Avila" <elavila@google.com>
Date: Tue, 6 Apr 2021 20:40:42 +0000
Subject: ANDROID: Add Image.lz4 to arm64 GKI outputs

Partners may want Image.lz4, so generate it as part of aarch64 builds.

Bug: 184667897
Signed-off-by: J. Avila <elavila@google.com>
Change-Id: I434287c881eb5cc906ff205e82866ede14014528
(cherry picked from commit c51e9f8bf84583e36d9fba18efb53cdd654a58f4)
---
 build.config.gki.aarch64 | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/build.config.gki.aarch64 b/build.config.gki.aarch64
--- a/build.config.gki.aarch64
+++ b/build.config.gki.aarch64
@@ -2,4 +2,12 @@
 . ${ROOT_DIR}/${KERNEL_DIR}/build.config.aarch64
 . ${ROOT_DIR}/${KERNEL_DIR}/build.config.gki
 
+MAKE_GOALS="${MAKE_GOALS}
+Image.lz4
+"
+
 GENERATE_VMLINUX_BTF=1
+
+FILES="${FILES}
+arch/arm64/boot/Image.lz4
+"
